### PR TITLE
Adds the trademark symbol to the ArcGis string

### PR DESCRIPTION
**Pull request by javierarce**
Adds :tm: symbol to the ArcGis string and updates the icon.

screen shot 2015-07-28 at 14 50 08

screen shot 2015-07-28 at 14 50 40

screen shot 2015-07-28 at 15 28 27

### CODE REVIEW

File: app/assets/stylesheets/common/tabs.css.scss [hunk 1]
```diff
@@ -65,7 +65,9 @@
     color: $cTypography-paragraphs;
   }
 }
-
+.TabLink.arcgis {
+  width: 130px;
```
Comment (javierarce) [inline app/assets/stylesheets/common/tabs.css.scss]: Increases the width of the element to account for the extra width of the new string.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5435/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5436/))

**Comment by javierarce** [comment]
@viddo, @josecruz: can you review code and design, please?

**Comment by viddo** [comment]
:+1:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5437/))

**Comment by josecruz** [comment]
`S` of Server needs to be on caps here:

ff44d6dc-3537-11e5-9f66-5f72d41c5e90

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5439/))

**Comment by javierarce** [comment]
Well spotted, @josecruz, but it seems I took the capture before I put the 's' in uppercase, so it's fixed in the code.

**Comment by josecruz** [comment]
Oh sry! Looks perfect to me then :+1:

**Pull request merged**